nca/NetworkConfig/LiveSim/istio_gateway/istio_gateway.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: istio-ingressgateway-livesim
```

Comment:
also add namespace here (for this pod resource), so that it is not in the default namespace